src/features/selection/js/selection.js

```diff
@@ -254,7 +254,7 @@
                  * @description Selects all visible rows.  Does nothing if multiSelect = false
                  * @param {Event} event object if raised from an event
                  */
-                selectAllVisibleRows: function (event) {
+                selectAllVisibleRows: function (evt) {
```

Comment:
This change is not necessary at all.

Reply:
Perhaps, but it would better comply with your coding style.

Reply:
It is purely cosmetic

Reply:
The change should be as small as possible, thats all :smile: